PowerShellTools.Test/ScriptDebuggerTest.cs

```diff
@@ -101,7 +106,11 @@ public void ShouldBreakOnBreakPoint()
                                new ScriptBreakpoint(null, fi.FullName, 3, 0, engineEvents.Object)
                            };
 
-            _debugger.SetBreakpoints(sbps);
+            _debugger.BreakpointManager.SetBreakpoints(sbps);

```

Comment:
I feel it is still convenient for testing purpose, it basically clear all existing bps, and set new ones all once, so I leave it there as a testing method for now.